[stable/spinnaker] Add a value for customizing service settings

legal90 opened the pull request:
Closes #12032, #11631, #12719, #13588, #13385  (as superseded)  

#### What this PR does / why we need it:

Using the newly added `additionalServiceSettings` value it's possible to override service-specific
parameters, which could not be done the custom profile.
For example, that might be used to override the version of some components, adding pod annotations, etc.
Read more: https://www.spinnaker.io/reference/halyard/custom/#custom-service-settings

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] ~Variables are documented in the README.md~ _This chart README doesn't contain the documentation of values_

legal90 committed:
[stable/spinnaker] Bump chart version

Signed-off-by: Mikhail Zholobov <legal90@gmail.com>

lwander reviewed (approved):
LGTM, not sure about the status of the e2e tests, that's outside of my domain

legal90 committed:
[stable/spinnaker] Add a value for customizing service settings

Using the newly added `additionalServiceSettings` value it's possible to override service-specific
parameters. That could be used to override some component versions or to add custom labels to pods.
Read more: https://www.spinnaker.io/reference/halyard/custom/#custom-service-settings

Signed-off-by: Mikhail Zholobov <legal90@gmail.com>
